docs/api/createAsyncThunk.md

```diff
@@ -136,6 +138,17 @@ interface RejectedAction<ThunkArg> {
   }
 }
 
+interface RejectedWithValueAction<ThunkArg, RejectedValue> {
```

Comment:
This is not really an existing type with that name, I'm not sure if we should document it that way.

Reply:
To be fair: none of these types actually exist. Should we have nonexistent interfaces interfaces in the documentation?

Reply:
There's certainly been times when I've written kinda hand-wave-y types in docs to illustrate a point, but if we do that it probably needs to be clarified.

Reply:
I'm not sure of the best language to use here. Open to suggestions.